internal/resolve/catalog.go

```diff
@@ -271,3 +299,18 @@ func CatalogWalker(
 		return nil
 	}
 }
+
+func isFBCEmpty(fbc *declcfg.DeclarativeConfig) bool {
+	if fbc == nil {
+		return true
+	}
+	return len(fbc.Packages) == 0 && len(fbc.Channels) == 0 && len(fbc.Bundles) == 0 && len(fbc.Deprecations) == 0 && len(fbc.Others) == 0
```

Comment:
Hi @joelanford 

IHMO IsEmpty should be implemented in DeclarativeConfig itself. However, that is not a blocker for us to move forward here.